### ISSUE TITLE
[BUG] Rancher doesn't fully count imported Harvester cluster

### ISSUE THREAD

**Issue by simonflood** [harvester/harvester#1511]
**Describe the bug**
When Rancher is first installed, on the dashboard it numbers and lists 1 cluster on it's home page - the local cluster.
After importing a Harvester cluster the numbers goes up to 2 but it still only lists the one cluster (local).
Clicking the hamburger menu lists one cluster (local) under 'EXPLORE CLUSTER'.
Clicking Cluster Management on the hamburger menu lists one cluster (local) but shows 2 to right of Clusters in left-hand menu.

**To Reproduce**
1. Install Harvester 0.3.0
2. Install Rancher 2.6.2
3. Import the Harvester cluster into Rancher
4. Check the Home and Cluster Management pages

**Expected behavior**
Rancher should include Harvester clusters in it's count of clusters and include them in the list of clusters. Clicking on the Harvester cluster name should take you to the appropriate Virtualization Management page.

**Support bundle**
N/A - whilst this might be an issue with the Harvester node driver it may actually be with Rancher.

**Environment:**
 - Harvester ISO version: 0.3.0
 - Underlying Infrastructure: Baremetal with Intel NUC
 - Rancher 2.6.2 on Raspberry Pi 4 Model B (4GB RAM) - have also seen with Rancher 2.6.1 on Harvester VM

**Additional context**
Two screenshots hopefully showing problem - I've marked discrepancies in red.
![Screenshot from 2021-11-04 16-21-30](https://user-images.githubusercontent.com/8300801/140378456-eb8570a3-ea4e-41a5-a68e-eb006fa25d82.png)
![Screenshot from 2021-11-04 16-15-29](https://user-images.githubusercontent.com/8300801/140378477-c529f60d-96d7-441f-a4fe-fd96b70d154f.png)

**Comment by guangbochen** [comment]
@simonflood How did u import the Harvester cluster into the Rancher, if u do it via the `Virtualization Management` page I would expect to see the cluster appears on that page.
https://docs.harvesterhci.io/v0.3/rancher/virtualization-management/#virtualization-management

The Cluster Management page is mainly for regular k8s clusters, so we decided to filter out the Harvester cluster.

**Comment by simonflood** [comment]
@guangbochen Yes my imported Harvester cluster appears on the VM page which is okay but I think it's confusing to include it in the count of clusters but then filter it out. I think it should be included in the count (so that's correct) but also listed as a cluster then when you click on a Harvester cluster it takes to you the appropriate VM sub-page.

Alternatively give us the option of including or filtering out Harvester clusters, perhaps with the count adjusting accordingly.

**Comment by wjimenez5271** [comment]
I would be concerned that that by counting it among our Kubernetes Clusters it sends the message that it is like the other k8s clusters in that list, when Harvester is not meant to be used as a typical workload cluster.
